### PR TITLE
Add crawler "inside" into scheduling

### PR DESCRIPTION
**Pull request by chenyang14**

### CODE REVIEW

File: server_api.py [hunk 1]
```diff
@@ -1019,3 +1019,50 @@ def get_upcoming_events(credentials):
...
+            sql = "INSERT INTO news " \
+                    +" (`data_type`, `serial_number`, `title`)" \
+                    +" VALUE (" \
+                    + str(news_data_type) + ", "\
+                    + "\"" + news_serial_number + "\", " \
+                    + "\"" + news_title + "\")"
+
+            try:
+                db.cmd(sql)
+            except DB_Exception as e:
```
Comment (Billy4195) [inline server_api.py]: This exception can be deleted? Because the line 1065 do the same thing.

File: arrange_schedule.py [hunk 2]
```diff
@@ -1188,6 +1189,60 @@ def crawler_google_drive_img(json_obj):
...
+            return return_msg
+
+        db.close()
+        return_msg["result"] = "success"
+        return return_msg
+    except DB_Exception as e:
+        db.close()
+        return_msg["error"] = e.args[1]
+        return return_msg
+
```
Comment (Billy4195) [inline arrange_schedule.py]: Too many useless blank lines.

File: news_crawler/news_crawler.py [hunk 3]
```diff
@@ -3,18 +3,44 @@
...
         link = post['href']
         title = post.text
-        #TODO add path in make_qrcode_image
-        serial_number = make_qrcode_image(link)
+        serial_number = make_qrcode_image(link,path)
+
+        send_obj = save_inside_db_data(serial_number, title)
+        news_insert_db(send_obj)
+            
+
```
Comment (Billy4195) [inline news_crawler/news_crawler.py]: Leave one blank line between two functions.

File: news_crawler/news_crawler.py [hunk 4]
```diff
@@ -35,3 +61,5 @@ def create_news_table():
 #create data_types for all websites crawler grabbed
 def create_news_data_types():
     create_data_type('inside')
+
```
Comment (Billy4195) [inline news_crawler/news_crawler.py]: These two lines are useless blank lines.

File: arrange_schedule.py [hunk 5]
```diff
@@ -1188,6 +1189,60 @@ def crawler_google_drive_img(json_obj):
         return_msg["error"] = e.args[1]
         return return_msg
 
+
+
```
Comment (Billy4195) [inline arrange_schedule.py]: Leave one blank line between two functions.

**Comment by chenyang14** [comment]
redundant lines and code mentioned above have been deleted

File: arrange_schedule.py (hunk 5 above)
Comment (Billy4195) [inline arrange_schedule.py]: Here are still some blank lines have not been deleted.

**Comment by Billy4195** [comment]
After finished the requested changes, please squash the commits after [140e987](https://github.com/SWLBot/electronic-blackboard/pull/47/commits/140e98761e1b52d056bd7bc52e22f88150ff2634) into 140e987

**Pull request merged**